Sources/TuistKit/Loader/Graph.swift

```diff
@@ -109,6 +110,21 @@ class Graph: Graphing {
             .filter({ $0.path == path })
             .map({ $0.target.name })
     }
+    
+    func staticLibraryDependencies(path: AbsolutePath, name: String) -> [DependencyReference] {
+        
+        guard let targetNode = self.targetNode(path: path, name: name) else {
+            return [ ]
+        }
+
+        return targetNode.dependencies
+            .compactMap{ $0 as? TargetNode }
+            .filter{ $0.target.product == .staticLibrary }
+            .map{ targetNode in
+                return DependencyReference.product(targetNode.target.productName)
+            }
+        
+    }
 
     func linkableDependencies(path: AbsolutePath, name: String) throws -> [DependencyReference] {
```

Comment:
I'd add a test to check if we are traversing the graph properly and returning the expected list of static libraries.

Reply:
Implemented `func test_trasistiveDependenciesForStaticLibraries()`